.profile.d/002_ssh_profile_setup.sh

```
@@ -0,0 +1,3 @@
+# .profile.d/002_ssh_profile_setup.sh
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)